source/_components/homekit_controller.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+title: "Homekit controller support"
+description: "Instructions how to integrate your Homekit devices within Home Assistant."
+date: 2018-03-19 21:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: apple-homekit.png
+ha_category: Hub
+ha_release: 0.66
+ha_iot_class: "Local Polling"
+---
+
+[HomeKit](https://developer.apple.com/homekit/) controller integration for Home Assistant allows you to connect HomeKit accessories to Home Assistant. This component should not be confused with the [HomeKit](homekit) component, which allows you to control Home Assistant devices via HomeKit.
+
+There is currently support for the following device types within Home Assistant:
+
+- [Light](../light.homekit_controller)
+
+The component will be automatically configured if the [`discovery:`](components/discovery/) component is enabled. For each detected Homekit accessory, a configuration prompt will appear in the web front end. Use this to provide the HomeKit PIN. It can be disabled by adding the following configuration:
+
+```yaml
+discovery:
+  ignore:
+    - homekit
```

Comment:
Should be call this `homekit_controller` or something? It can easily confused with `homekit` component

Reply:
yes, agree

Reply:
This is the name that comes from netdisco, so we can't change that without changing the behaviour of netdisco, and that might break other users of netdisco. I went both ways on the component name - it's implementing a homekit controller but it's for controlling homekit accessories, so I could switch that.

Reply:
If I may interject my 2 cents as a person who frequently helps on the forums. Now that there’s two HomeKit components they should really both be renamed. The current one to homekit_bridge (as that’s what it emulates) and this to homekit_accessories (since that’s what it supports adding to HA). There’s already confusion re the differences between Homekit and Homebridge, users can’t tell them apart since they both expose devices to HomeKit.